### ISSUE TITLE
is it possible to Install this module without installing nginx from source?

### ISSUE THREAD

**Issue by Ajar-Ajar** [yaoweibin/nginx_tcp_proxy_module#125]
Hi,
Trying to follow your installation instructions and install nginx from source does not work in my case.
Is it possible to use "apt-get install nginx"  and after installing it using apt-get - installing nginx_tcp_proxy_module?

Thank you

**Comment by Jefffish09** [comment]
@Ajar-Ajar Yep, I agree with you. “apt-get” is more convenient. Additionally, now is 2019, do you have any idea to solve this issue?
